src/applications/personalization/profile360/vet360/util/local-vet360.js

```diff
@@ -5,7 +5,8 @@ export function isVet360Configured() {
   return [
     'staging.vets.gov',
     'www.vets.gov',
-    'preview.va.gov'
+    'preview.va.gov',
+    'staging.va.gov'
```

Comment:
Can we also add `va.gov` and `www.va.gov` to the list? I feel like we'll definitely forget this again otherwise lol.